recipe/meta.yaml

```diff
@@ -28,19 +28,24 @@ source:
       - support_new_cuda_layout.diff  # [(linux and cuda_compiler_version == "12.0") or (build_platform != target_platform)]
 
 build:
-  number: 2
+  number: 3
   # TODO: turn on win64 + CUDA 12 once it's ready
   # TODO: turn on aarch64 + CUDA 12 once glibc 2.28 is ready
   skip: true  # [cuda_compiler_version == "None" or (cuda_compiler_version == "12.0" and not linux64)]
   script:
-    - export CUPY_NVCC_GENERATE_CODE="arch=compute_60,code=sm_60;arch=compute_70,code=sm_70;arch=compute_80,code=sm_80"  # [aarch64]
+    - export CUPY_NVCC_GENERATE_CODE="arch=compute_60,code=sm_60;arch=compute_70,code=sm_70;arch=compute_80,code=sm_80"  # [aarch64 or ppc64le]
```

Comment:
It's not a potential problem, it's a real problem and I really want to avoid hitting timeouts after straight 6-hr CI runs. That's why we switched to cross-compiling, but unfortunately [it's no longer working since the last weekend due to out of space](https://github.com/conda-forge/cupy-feedstock/pull/208#issuecomment-1575315961). I unfortunately don't have time to investigate potential patches this week, so this is only a stop-gap solution.

CC86 can be supported by JIT'ing too, so it's only a nice-to-have.

Reply:
Totally understand. By “potential” I only meant that it sometimes passed.

Sounds like dropping earlier arches 35-50 may have made the compile time better since the last time that qemu was used. If time allows on some future PR, it’d be nice to add 86, but it’s definitely a tradeoff for compile time…

Reply:
Maybe we can leave a comment mentioning why the override is used on a future PR. No need to rerun CI for that reason alone.

Reply:
Added in #208.